runbook/callback_context_reader_windows.go

```diff
@@ -0,0 +1,41 @@
+//go:build windows
+// +build windows
+
+package runbook
+
+import (
+	winio "github.com/Microsoft/go-winio"
+	"github.com/sirupsen/logrus"
+	"net"
+)
+
+var CallbackContextReaderFunc = CallbackContextReaderForWindows
```

Comment:
you can define a CallbackContextHandler interface and implement it for linux and windows. then you can initialize the correct struct once in the runbook executor